papaya-core/src/main/java/pl/edu/agh/papaya/service/project/ProjectService.java

```diff
@@ -57,6 +61,17 @@ public boolean isUserInProject(Project project, String userId) {
                 .anyMatch(userId::equals);
     }
 
+    public boolean isUserInProject(Project project, User user) {
+        return isUserInProject(project, user.getId().toString());
+    }
+
+    public boolean hasUserAdminRights(Project project, User user) {
```

Comment:
`userHasAdminRights`?

Reply:
`isAdmin`?

Reply:
Chociaż u @kjarosh mamy takie coś w `Project`, także to pewnie i tak wyleci, bo w projekcie chyb a ładniej? Ewenualnie tu można chamsko ukraść
```
    public boolean isAdmin(User user) {
        return hasRole(user, UserRole.ADMIN);
    }

    public boolean hasRole(User user, UserRole role) {
        return usersInProject.stream()
                .filter(up -> up.getUser().equals(user))
                .findAny()
                .map(UserInProject::getUserRole)
                .map(role::equals)
                .orElse(false);
    }
```